Configuration/odin-views.json

```diff
@@ -50,21 +50,23 @@
                     }
                 },
                 "unsecure": {
-                    "view": "fost.response.401",
+                    "view": "fost.view.pathprefix",
                     "configuration": {
-                        "schemes": {
-                            "Bearer": {}
-                        },
-                        "view": "fost.view.pathprefix",
-                        "configuration": {
-                            "": "odin/unsecure/sql",
-                            "me/": "fost.response.403",
-                            "me/password": "odin.password.me",
-                            "forgotten-password/": "odin/forgotten-password",
-                            "user/": {
-                                "view": "odin.user.unsecure",
-                                "configuration": "odin/unsecure/sql"
+                        "": {
+                            "view": "fost.response.401",
+                            "configuration": {
+                                "schemes": {
+                                    "Bearer": {}
+                                },
+                                "view": "odin/unsecure/sql"
                             }
+                        },
+                        "me/": "fost.response.403",
+                        "me/password": "odin.password.me",
+                        "forgotten-password/": "odin/forgotten-password",
+                        "user/": {
```

Comment:
This view configuration looks totally broken to me. I don't think it does anything at all does it? Could just delete it.